requirements-test.txt

```diff
@@ -0,0 +1 @@
+pytest==6.2.5
```

Comment:
Poetry might make more sense long term for managing/packaging this but went with plain pip for now to be easier :)